Extensions/PlatformBehavior/tests/platformerobjectruntimebehavior.spec.js

```diff
@@ -930,10 +1015,31 @@ describe('gdjs.PlatformerObjectRuntimeBehavior', function () {
         const lastY = object.getY();
         runtimeScene.renderAndStep(1000 / 60);
         expect(object.getBehavior('auto1').isOnLadder()).to.be(true);
+        //TODO Probably a bug, uncomment it after it's fixed
+        //expect(object.getBehavior('auto1').isMoving()).to.be(false);
```

Comment:
You mean `isMoving()` is `true` when you're on a ladder and not doing anything? Sounds weird.

Reply:
It was just the inertia of the right key in a test.